nativeauthenticator/templates/change-password.html

```diff
@@ -25,23 +31,38 @@
         <h1>
             Change password for {{user_name}}
         </h1>
+        
+        <p>Please enter your current password and the new password you want to set it to. If you have forgotten your password, an admin can reset it for you.</p>
 
         <div class="form-group">
-            <label for="password_input">New password:</label>
-            <div class="input-group">
-                <input id="password_input" type="password" name="password" val="" autocapitalize="off" autocorrect="off" class="form-control" />
+            <label for="password_input">Old password:</label>
+            <div class="input-group col-xs-6">
+                <input id="password_prove" type="password" name="old_password" val="" autocapitalize="off" autocorrect="off" class="form-control" />
```

Comment:
I suggest we favor consistency of the names here, so we let the id be `old_password_input` or similar instead of creating a new name for this.

Reply:
See above, I made the names consistent to `"old_password_input"`, `"new_password_confirmation_input"` and `"new_password_confirmation_input"`.